lib/broker.go

```diff
@@ -0,0 +1,142 @@
+package sebak
+
+import (
+	"io"
+)
+
+type EventType string
+
+const (
+	EventDummy  EventType = "dummy"
+	EventDummy2           = "dummy2"
+)
```

Comment:
Why is this needed? `dummy` looks like test purpose.

Reply:
Event types are not yet defined. Hence, I added those two.
Maybe those are changed to practical names